src/test/java/net/datafaker/providers/base/DateAndTimeTest.java

```diff
@@ -15,18 +15,32 @@
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAccessor;
 import java.time.zone.ZoneRules;
 import java.util.Date;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 
+import static java.time.Duration.ZERO;
+import static java.time.temporal.ChronoUnit.DAYS;
+import static java.time.temporal.ChronoUnit.HOURS;
+import static java.time.temporal.ChronoUnit.MICROS;
+import static java.time.temporal.ChronoUnit.MILLIS;
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static java.time.temporal.ChronoUnit.NANOS;
+import static java.time.temporal.ChronoUnit.SECONDS;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /**
  * @author pmiklos
  */
+@SuppressWarnings("removal")
```

Comment:
If something here is deprecated for removal why not address the issue now instead of waiting for it to fail in the future?

Reply:
The deprecated class is DataFaker's own class `DateAndTime.java`.
And this `DateAndTimeTest.java` is the test for this deprecated class.  :)
We still need this test until we finally delete both files.

Reply:
Oh okay, I thought it was something in the Java base. Thanks for clarifying.